metrics/sinks/influxdb/influxdb.go

```diff
@@ -107,7 +111,11 @@ func (sink *influxdbSink) ExportData(dataBatch *core.DataBatch) {
 
 			dataPoints = append(dataPoints, point)
 			if len(dataPoints) >= maxSendBatchSize {
-				sink.sendData(dataPoints)
+				sink.wg.Add(1)
+				sink.conChan <- struct{}{}
```

Comment:
add an inline comment here: `// use the channel to block until there's less than the maximum number of concurrent requests running`

Reply:
Will do.